generator/explores/operational_monitoring_explore.py

```diff
@@ -45,16 +48,57 @@ def from_dict(
         """Get an instance of this explore from a dictionary definition."""
         return OperationalMonitoringExplore(name, defn["views"], views_path, defn)
 
-    def _to_lookml(self, v1_name: Optional[str]) -> List[Dict[str, Any]]:
+    def _to_lookml(
+        self,
+        bq_client: bigquery.Client,
+        v1_name: Optional[str],
+        data: Dict = {},
+    ) -> List[Dict[str, Any]]:
+        base_view_name = self.views["base_view"]
+
+        namespace_data = data.get("compute_opmon_dimensions", {}).get(
+            base_view_name, {}
```

Comment:
Instead of having to define `data_functions` and passing around `namespace_data`, I'm wondering if it is possible to move `compute_opmon_dimensions()` into the `OperationalMonitoringExplore` implementation. The explore does have access to all the `view` names and could pass them into `compute_opmon_dimensions()`.
The `compute_opmon_dimensions()` is pretty specific to operational monitoring, so it's unlikely we will want to reuse it elsewhere. Moving it out of `lookml_utils` (which mostly contains generic lookml functions) might be nice.

Reply:
The `namespace_data` gets passed into both the explore and the dashboard so I'm not sure `OperationalMonitoringExplore` is the right place for `compute_opmon_dimensions()`.

I think there could be a separate `utils` file for opmon so that `compute_opmon_dimensions()` doesn't need to stay in `lookmk_utils`. Since the data produced by that function is used by both explores and dashboards, I think it doesn't necessarily belong in either.

Reply:
I see. I think this makes sense. I'm not super in love with having a new `data_functions` attribute, but it's okay for now. 

As we are moving towards generating `aggregate_tables` for other generated explores, we might want to refactor this and have a better way of making dimensions available across views, explores and dashboards (+ some way to filter/transform them). I'll write up an issue to keep track of this.

For now moving `compute_opmon_dimensions()` into a separate file would be great.